src/main/kotlin/com/team4099/robot2022/config/Constants.kt

```diff
@@ -68,11 +68,13 @@ object Constants {
     val MAX_AUTO_ANGULAR_ACCEL = 90.0.degrees.perSecond.perSecond
 
     const val ABSOLUTE_GEAR_RATIO = 1.0
+    const val DRIVE_SENSOR_CPR = 2048 // TODO
+    const val DIRECTION_SENSOR_CPR = 2048 // TODO
     const val DRIVE_SENSOR_GEAR_RATIO = (12.0 / 21.0) * (15.0 / 45.0)
```

Comment:
pls also correct the gear ratios here

https://www.swervedrivespecialties.com/products/mk4i-swerve-module

we're running the L2 ratios for drive